src/org/usfirst/frc4904/robot/humaninterface/operators/DefaultOperator.java

```diff
@@ -0,0 +1,31 @@
+package org.usfirst.frc4904.robot.humaninterface.operators;
+
+
+import org.usfirst.frc4904.robot.RobotMap;
+import org.usfirst.frc4904.robot.commands.IndexerGrabberClasp;
+import org.usfirst.frc4904.robot.commands.IntakeRollersIntake;
+import org.usfirst.frc4904.robot.commands.IntakeRollersOuttake;
+import org.usfirst.frc4904.robot.commands.IntakeSquared;
+import org.usfirst.frc4904.robot.commands.OuttakeSquared;
+import org.usfirst.frc4904.standard.humaninput.Operator;
+
+public class DefaultOperator extends Operator {
+	public DefaultOperator() {
+		super("DefaultOperator");
+	}
+
+	public DefaultOperator(String name) {
+		super(name);
+		// TODO Auto-generated constructor stub
+	}
+
+	@Override
+	public void bindCommands() {
+		RobotMap.Component.joystick.button3.onlyWhileHeld(new IntakeSquared());
+		RobotMap.Component.joystick.button3.whenReleased(new IndexerGrabberClasp());
+		RobotMap.Component.joystick.button4.onlyWhileHeld(new OuttakeSquared());
+		RobotMap.Component.joystick.button4.whenReleased(new IndexerGrabberClasp());
+		RobotMap.Component.joystick.button5.onlyWhileHeld(new IntakeRollersIntake());
+		RobotMap.Component.joystick.button6.onlyWhileHeld(new IntakeRollersOuttake());
```

Comment:
for now, delete the mappings to buttons 5 & 6 until we decide exactly how much manual control is needed/wanted.

Reply:
what no. we should have all the manual control we can until we figure out how we want it to be configured for actual driving. better to be able to move it than to not